### ISSUE TITLE
Installation instructions are out of date

### ISSUE THREAD

**Issue by raphj** [tdewolff/minify#319]
Hello,

I am installing minify on openSUSE.

The following instructions from the README do not seem up to date:
```sh
git clone https://github.com/tdewolff/minify.git
cd minify/cmd/minify
./install.sh
```
I don't have an `install.sh` script here.

Thanks for this project!

**Comment by tdewolff** [comment]
Thanks for the heads up, should be fixed now! Let me know if you run into any trouble.

**Comment by raphj** [comment]
Oh, that's where it went then!

You know, I'm not familiar at all with go. I only knew that go generates self sufficient binaries. I did not figure out where `make install` (`go install`, for that matter) put the resulting binary. I ended up reading the Makefile, running `make release`, seeing it to generate all the versions (good job by the way, this is inspiring), opening the generated tarball and copying the binary into my `PATH`.

I tried to run your new installation instructions and can confirm they work :-)
